core/src/main/java/org/elasticsearch/common/unit/ByteSizeValue.java

```diff
@@ -32,11 +32,11 @@
 public class ByteSizeValue implements Writeable {
 
     private final long size;
-    private final ByteSizeUnit sizeUnit;
+    private final ByteSizeUnit unit;
 
     public ByteSizeValue(StreamInput in) throws IOException {
         size = in.readVLong();
-        sizeUnit = ByteSizeUnit.BYTES;
+        unit = ByteSizeUnit.BYTES;
```

Comment:
I'm changing this one thing at a time - next will be ByteSizeValue serialization. So if you're OK I'd like to keep this as it is for now and change it in a follow up PR.